src/app.js

```diff
@@ -277,24 +286,7 @@ var run = function () {
     }).appendTo('body')
   })
 
-  // ---------------- tabbed menu ------------------
-  $('#options li').click(function (ev) {
-    var $el = $(this)
-    selectTab($el)
-  })
-
-  var selectTab = function (el) {
```

Comment:
that's cool to remove `selectTab` from here but it is used. Look at the function `startdebugging` in app.js. we could pass the debugger event to ´tabbedMenu´. and listen on `startDebugging` event
.